music21/configure.py

```diff
@@ -1762,15 +1773,15 @@ def run():
     else:
         # only if running tests
         t = Test()
-        te = TestUserInput()
+        te = TestExternal()
 
         if len(sys.argv) < 2 or sys.argv[1] in ['all', 'test']:
             import music21
             music21.mainTest(Test)
 
         # arg[1] is test to launch
         elif sys.argv[1] == 'te':
-            # run test user input
+            # run test external
```

Comment:
I suppose this shouldn't be `'te'` anymore.

Reply:
Sorry, I don't understand the comment.